eve/main.yml

```diff
@@ -2073,45 +2086,44 @@ stages:
       - ShellCommand:
           <<: *multi_node_fast_tests
           name: Run fast tests on Bastion after certificates renewal
-      - ShellCommand:
-          <<: *generate_report_over_ssh
-          env:
-            <<: *_env_generate_report_over_ssh
-            HOSTS_LIST: "bootstrap node-1"
+      - ShellCommand: *generate_report_over_ssh
       - ShellCommand:
           <<: *collect_report_over_ssh
           env:
             <<: *_env_collect_report_over_ssh
-            HOSTS_LIST: "bootstrap node-1"
-            STEP_NAME: multiple-nodes-centos
+            STEP_NAME: "multiple-nodes-%(prop:os:-centos-7)s"
       - Upload: *upload_report_artifacts
       - ShellCommand:
           <<: *add_final_status_artifact_success
           env:
             <<: *_env_final_status_artifact_success
             <<: *_install_multi-node_junit_info
-            STEP_NAME: multiple-nodes-centos
+            STEP_NAME: "multiple-nodes-%(prop:os:-centos-7)s"
       - Upload: *upload_final_status_artifact
       - ShellCommand:
           <<: *wait_debug
           timeout: 14400
           env:
-            STEP_NAME: multiple-nodes-centos
+            STEP_NAME: "multiple-nodes-%(prop:os:-centos-7)s"
             DURATION: "14400"
-      - ShellCommand:
-          <<: *terraform_destroy
-          env: *_env_terraform_multi_node
+      - ShellCommand: *terraform_destroy
 
   bootstrap-restore:
     _metalk8s_internal_info:
       junit_info: &_bootstrap_restore_junit_info
         TEST_SUITE: install
-        CLASS_NAME: multi node.centos7
+        CLASS_NAME: "multi node.%(prop:os:-centos-7)s"
         TEST_NAME: bootstrap restore
       simultaneous_builds: 20
     worker: *terraform_worker
     steps:
       - Git: *git_pull
+      - SetProperty:
+          # NOTE: Set property will be skipped if the value is already set,
```

Comment:
Btw it's not the normal behavior of "SetProperty" but since I do a `value: "%(prop:nodes_count:-2)s"` yes it's always the case

Or yes you right it's not really "skipped" just we take the current value. I will update the comment message in another PR

Reply:
> since I do a value: "%(prop:nodes_count:-2)s" yes it's always the case

Ah right I missed this :+1: no rush for updating the comment :wink: